src/store/CategoricalDisplayStore.js

```diff
@@ -198,12 +198,8 @@ export default class CategoricalDisplayStore {
     }
 
     @action.bound clearFilterText() {
-        this.filterText = '';
+        this.setFilterText('');
         this.searchInput.value = '';
-        setTimeout(() => {
-            this.scroll.refresh();
-            this.updateScrollSpy();
-        }, 0);
```

Comment:
does the active category update when you remove this? Did you check?

Reply:
@brucebinary  yeah, it get update when I click `times icon` or `clear search text`

Reply:
@brucebinary  check my last changes, I double check with them, and all work fine